extensions/3DTILES_implicit_tiling_multiple_contents/README.md

```diff
@@ -0,0 +1,174 @@
+# 3DTILES_implicit_tiling_multiple_contents
+
+## Contributors
+
+* Sean Lilley, Cesium
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the 3D Tiles 1.1 specification.
+
+## Optional vs. Required
+
+This extension is required, meaning it must be placed in both the `extensionsUsed` and `extensionsRequired` lists in the tileset JSON.
+
+## Overview
+
+This extension introduces a new [tile metadata semantic](https://github.com/CesiumGS/3d-tiles/blob/main/specification/Metadata/Semantics/README.adoc#tile) `TILE_CONTENT_COUNT` for indicating the number of contents referenced by each tile in an implicit tileset. This is more efficient than the existing [multiple contents approach](https://github.com/CesiumGS/3d-tiles/tree/main/specification/ImplicitTiling#multiple-contents) when tiles have a widely varying number of contents.
```

Comment:
Can you mix this extension with the existing approach to handling multiple contents?